packages/transport/src/modules/relationships/RelationshipsController.ts

```diff
@@ -155,6 +155,17 @@ export class RelationshipsController extends TransportController {
             throw this.newCacheEmptyError(RelationshipTemplate, template.id.toString());
         }
 
+        const queryForExistingRelationship = {
+            "peer.address": template.cache.createdBy.toString(),
+            status: { $in: [RelationshipStatus.Pending, RelationshipStatus.Active, RelationshipStatus.Terminated, RelationshipStatus.DeletionProposed] }
+        };
+
+        const existingRelationshipToPeer = await this.getRelationships(queryForExistingRelationship);
+
+        if (existingRelationshipToPeer.length !== 0) {
+            throw CoreErrors.relationships.relationshipAlreadyExists(existingRelationshipToPeer[0].status);
+        }
```

Comment:
you could put this code to a canCreateRelationship method for re-use! But you can also do that in another PR.